src/routes/profile-route/profile-fests-route.jsx

```diff
@@ -10,23 +10,25 @@ import {
   Keyword,
   SlopFestModal,
 } from "../../components/index";
-import { GET_USER_FESTS } from "../../graphql/get-user-fests";
+import { GET_FEST, GET_USER_FESTS, UPDATE_FEST } from "../../graphql";
 import { useCurrentUser, useModals } from "../../hooks";
-import checkMark from "../../images/check-mark-dark.svg";
+import checkMarkBlack from "../../images/check-mark-dark.svg";
 import checkMarkWhite from "../../images/check-mark.svg";
 import { ProfileHorizontalMenu, ProfileSidebar } from "./index";
 
 export const ProfileFestsRoute = () => {
   const { currentUser } = useCurrentUser();
+
   const currentDate = new Date().toLocaleString("default", {
```

Comment:
currentDate can be removed, since I'm assuming its original purpose was to compare dates for future or past fests. You accomplished this with dayjs however!